### ISSUE TITLE
Dev: fix Error: Missing Button on FAQ page

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/tdm-calculator#3041]
### Overview
We need to fix the Error: Missing Button on FAQ so that page can be WCAG compliant.

#### Details
Category: Error
Name: Missing Button
(The Accessibility wiki for this Error has a solution)

Page that will be worked on:
FAQ

### Action Items
- [ ] Review wiki page: Accessibility, How to run the WAVE browser extension (Resource 1.01)
   - [ ] Install the WAVE browser extension
- [ ] Log in as the user indicated (resource 1.02.01) 
- [ ] navigate to the page you are going to fix (resource 1.02.02)
- [ ] Open the TDM WAVE Accessibility Audit Spreadsheet, for this Error (resource 1.03) and find the row that relates to this  issues page.
- [ ] Run the WAVE extension with no elements focused, or clicked
- [ ] Find the Error: Missing Button referenced in the Spreadsheet (1.03)
- [ ] This Error  has been resolved on other pages, so review `Project Team Solution` section (resource 1.04.01.01)
- [ ] Apply solution
- [ ] Submit PR
- [ ] Add PR to spreadsheet in PR column next to Error (resource 1.03)
- [ ] On this issue
   - [ ] Add labels: 
      - `ready for accessibility lead`
   - [ ] Change status to: `Technical Review`
   - to the epic #2892
     - [ ] Add the label `ready for product`
      - [ ] Add this comment 
         ```
          - #3041 is closed.  Please review to see if there are any remaining instances of this problem
         ```

### Resources/Instructions
- 1.01[How to run the WAVE browser extension](https://github.com/hackforla/accessibility/blob/main/docs/getting-started/wave-browser-extension.md)
- 1.02 Application details
   - 1.02.01 User Profile Required: visitor (not logged in)
   - 1.02.02 Page URL: https://tdm-dev.azurewebsites.net/faqs
      - Requires sign in: FALSE
      - Can access page directly from URL: TRUE 
- 1.03 [TDM WAVE Accessibility Audit Spreadsheet, tab: 2. Audit Log, filter: Error: Empty button](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0&fvid=1522502955)
   -  Spreadsheet Row: 38
- 1.04 [Accessibility Known Issues Index](https://github.com/hackforla/accessibility/blob/main/docs/known-issues-and-solutions/solutions-index/index.md)
    - 1.04.01 [WAVE Known Accessibility Issue: Error - Empty Button](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/error-empty-button)
        - 1.04.01.01 [WAVE Known Accessibility Issue: Error - Empty Button, Project Team Solution](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/error-empty-button/#project-team-solution)

Additional reading (optional)
- https://hackforla.github.io/accessibility/

#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - [Staging Deck, slide 3041](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g39673b86911_4_35#slide=id.g39673b86911_4_35)
- Release deck
   - [2026-MM-DD Release deck, slide 3041]

**Comment by Rabia2219** [comment]
@geolunalg 
Please add a level label. You can choose from one of the following:

- `good first issue`
- `level: easy`
- `level: medium`
- `level: hard`

Once you have added the level label, please:
- remove the `level: missing` label
- remove the `ready for accessibility lead` label
- add the `ready for prioritization` label
